README.md

```diff
@@ -0,0 +1,42 @@
+A library that explores dependencies from a given module or namespace in a local Python environment, then classifies these dependencies.
```

Comment:
`from a given module or namespace`
it's always a namespace, isn't it? module can be part of namespace